src/Components/ArticlesList.jsx

```diff
@@ -35,7 +37,9 @@ const ArticlesList = () => {
     <div className="articles-list-container">
       <div className="article-cards-container">
         {articles.map((article) => (
-          <ArticleCard key={article.article_id} article={article} />
+             <Link className = 'article-link' key={article.article_id} to={`/articles/${article.article_id}`}>
+             <ArticleCard article={article} />
```

Comment:
I'd have your article card render the relevant image on the page to make it easier for the user.